server/filters/recipientTableRowsFilter.ts

```diff
@@ -9,7 +10,9 @@ export default function recipientTableRowsFilter(recipients: Array<Recipient>):
   return recipients.map((recipient, idx) => {
     return Array.of(
       { text: recipient.prisonerName }, // Column 1 - Prisoner Name
-      { text: recipient.prisonNumber }, // Column 2 - Prison number or DOB - TODO SLM-81
+      {
+        text: recipient.prisonNumber ? recipient.prisonNumber : moment(recipient.prisonerDob).format('DD-MM-YYYY'), // Column 2 - Prison number or DOB
+      },
```

Comment:
Can prisoner DOB ever be absent / null?

Reply:
Apparently it does: https://caniuse.com/autofocus

Reply:
Yes we capture prisoner name and either prison ID or prisoner DOB. With a separate form for each method bizarrely :shrug: